### PR TITLE
add `delete-only-untagged-versions` param

### PR DESCRIPTION
**Pull request by s-anupam**
Adds a new boolean parameter to action `delete-only-untagged-versions`. When true, only versions that do not have any tags will be deleted. This parameter is only applicable to `container` type packages and ignored for all other package types. Defaults to false

- Fixes https://github.com/actions/delete-package-versions/issues/87

### CODE REVIEW

**Comment by sbe-arg** [comment]
Thank you for doing this!

File: README.md [hunk 1]
```diff
@@ -375,11 +401,14 @@ This action deletes versions of a package from [GitHub Packages](https://github.
       token: ${{ secrets.PAT }}
   ```
 
+# GitHub Enterprise Server
```
Comment (s-anupam) [inline README.md:404]: Just adding this section here. Missed adding in the last PR https://github.com/actions/delete-package-versions/pull/91

**Review by nishthaGupta — APPROVED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -375,11 +401,14 @@ This action deletes versions of a package from [GitHub Packages](https://github.
...
+
+This action works with GitHub Enterprise Server. It uses the `GITHUB_API_URL` environment variable to determine the GitHub Enterprise Server URL. The environment variable is automatically set in workflows. GitHub Connect should be configured on the server to allow using the action from dotcom. See - [Enabling automatic access to GitHub.com actions using GitHub Connect](https://docs.github.com/en/enterprise-server/admin/github-actions/managing-access-to-actions-from-githubcom/enabling-automatic-access-to-githubcom-actions-using-github-connect)
```
Comment (tinaheidinger) [inline README.md:406]: lgtm, just use "GitHub.com" instead of dotcom

**Pull request merged**